codebase/superdarn/src.idl/lib/main.1.25/snd.pro

```diff
@@ -2,7 +2,24 @@
 ; =======
 ; Author: E.G.Thomas
 ;
-;
+; 
+; RST is free software: you can redistribute it and/or modify
+; it under the terms of the GNU General Public License as published by
+; the Free Software Foundation, either version 3 of the License, or
+; (at your option) any later version.
+; 
+; This program is distributed in the hope that it will be useful,
+; but WITHOUT ANY WARRANTY; without even the implied warranty of
+; MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+; GNU General Public License for more details.
+; 
+; You should have received a copy of the GNU General Public License
+; along with this program. If not, see <https://www.gnu.org/licenses/>.
+; 
+; Modifications:
+; 
+; Functions
```

Comment:
```suggestion
```

This extra "Functions" label should not have been inserted here.

Reply:
Is this the only file in the directory that needs this change?